mission/finite_state_machine/scripts/fsm_helper.py

```diff
@@ -8,22 +8,26 @@
 from vortex_msgs.msg import MoveGoal, MoveAction
 from tf.transformations import quaternion_from_euler
 
-
-move_action_server = '/controller/move'
+move_action_server = '/guidance/move'
+# maybe create class for this? 
+# something to signal that the **_move functions are used for crosstalk
+# between the FSM and the Guidance systems.
+# rename the file accordingly too; put in own folder
 
 def dp_move(x, y, z=-0.5, yaw_rad=0):
     goal = MoveGoal()
 
-    goal.guidance_type = 'DP'
+    goal.guidance_type = 'PositionHold'
     goal.target_pose.position = Point(x, y, z)
     goal.target_pose.orientation = Quaternion(*quaternion_from_euler(0, 0, yaw_rad))
```

Comment:
Why change from 'DP' to 'PositionHold'?

Reply:
Felt like it was more descriptive for the usage of dp_move at the time. Not really sure what naming convention is the best,  and whether or not it will change as the fsm is expanded